docs/docs/addons-create-fragment.md

```diff
@@ -4,8 +4,8 @@ title: Keyed Fragments
 permalink: docs/create-fragment.html
 layout: docs
 category: Add-Ons
-prev: animation.html
-next: perf.html
```

Comment:
Because it was incorrect, as the diff in the other PR points out and fixes. When I cherry-picked this to make a stand-alone branch/PR I either kept this correction or made it again because it was convenient.